### ISSUE TITLE
Patcher: explain the mono option better

### ISSUE THREAD

**Issue by Taverius** [ryanoasis/nerd-fonts#165]
Nowhere does it say you need it for Mac/Win GVim to pick up the fonts :)

Had me scratching my head for an hour or so, looking at various topics on SO, before I happened on the a closed issue that mentions this.

**Comment by ryanoasis** [comment]
@Taverius Sorry about that.

We tried to explain better in some of the markdown files, for example: [AnonymousPro](https://github.com/ryanoasis/nerd-fonts/blob/master/patched-fonts/AnonymousPro/font-info.md)

Can I ask which closed issue you are referring to? :blush: 

> you need it for Mac/Win GVim to pick up the fonts

Hmm it shouldn't be required _but_ lately with the GUI builds of Vim there is an ambiguous width issue (glyphs cut in half). What was the issue? The fonts weren't working at all unless you used the mono versions?

> explain the mono option better

I am all for this :smile:. Did you have any specific place where more info would have helped? The readme? The patcher (`--help`) ? Somewhere else?

**Comment by Taverius** [comment]
> Hmm it shouldn't be required but lately with the GUI builds of Vim there is an ambiguous width issue (glyphs cut in half). What was the issue? The fonts weren't working at all unless you used the mono versions?

The latter, now working at all, GVim just wasn't seeing them - I'm running a very recent 8.0.

So its a 'lately' thing? Hopefully upstream is aware and *someone* who understands the codebase can do something about it.

As for explaining, the `--help` *sounds* like a decent place for such a hint - about it being sometimes useful to get certain editors to see the font on mac/win - but it also might make the option text too bloated ... up to you.

I'm just trying to save some time for anyone else who wants to patch Consolas/Input on non-(li|u)nix.

**Comment by RuRo** [comment]
In my opinion, the correct place for such explanations is the README.md. There are on the other hand things in the README that shouldn't be there ~cough~images with wrong character codes~cough~.
If nobody does anything about this before the 27th, I can try to 'refactor' the Readme.md

**Comment by ryanoasis** [comment]
@RuRo I agree readme would best.

> images with wrong character codes

Do you mean the "Seti-UI + Custom" set and "Font Awesome Extension" set? :blush: 

> If nobody does anything about this before the 27th, I can try to 'refactor' the Readme.md

Sure I am always open to PRs :smile: 

Honestly I haven't made much progress lately on getting the next release (v1.1) out the door.. but I don't think there is much left to do. Possibly this weekend but I'd hate to promise :blush:

**Comment by luisdavim** [comment]
More info on the README would be great. 
For example, I have no idea what the difference between variations, like with these:
```
Ubuntu Mono Nerd Font Complete Mono Windows Compatible.ttf
Ubuntu Mono Nerd Font Complete Mono.ttf
Ubuntu Mono Nerd Font Complete Windows Compatible.ttf
Ubuntu Mono Nerd Font Complete.ttf
Ubuntu Mono derivative Powerline Nerd Font Complete Mono Windows Compatible.ttf
Ubuntu Mono derivative Powerline Nerd Font Complete Mono.ttf
Ubuntu Mono derivative Powerline Nerd Font Complete Windows Compatible.ttf
Ubuntu Mono derivative Powerline Nerd Font Complete.ttf
```

**Comment by ryanoasis** [comment]
@luisdavim Yes you are right.

This should now mostly be taken care of now with: https://github.com/ryanoasis/nerd-fonts/commit/b5e827191482b1e0ddcc8cf3943341d6ce9ea531 and https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting#tldr

**Comment by luisdavim** [comment]
thanks. and what about the fonts that have powerline in their names? They all seem to have the powerline glyphs anyway...

**Comment by ryanoasis** [comment]
Yeah good point. Those are originally from https://github.com/powerline/fonts

Either we should upgrade the unpatched fonts to remove the 'for Powerline'/'powerline' or filter that out in the patcher.

**Comment by ryanoasis** [comment]
Considering this closed (with v1.1.0 release)
